www/docs/src/content/docs/start-here/getting-started.mdx

```diff
@@ -19,6 +16,12 @@ To start using Astro StudioCMS, you'll need:
 
 ## Creating an Astro Studio account
 
+:::note
+Astro Studio is currently in closed beta. You'll need an invite to create an account.
+
+While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```

Comment:
```suggestion
We're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again and will be informing everyone through our [Discord channel](https://chat.astro-studiocms.xyz) when signups for Astro Studio are open again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```

f that, we aint customer support.

Reply:
> we aint customer support.

That is in fact wrong sir...  i spent many years as IT support.  so dont hate on support.